plugins/rebalance/rest.go

```diff
@@ -44,14 +44,23 @@ func rebalanceStartHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	rebalinfo := createRebalanceInfo(volname, &req)
-	if rebalinfo == nil {
-		logger.WithError(err).Error("failed to create Rebalance info")
+	rebalInfo, err := GetRebalanceInfo(volname)
```

Comment:
are you missing error handling here?

Reply:
No I want the error to be not nil

Reply:
what happens when we get ```err !=nil``` don't we need to stop proceeding further
and return the error to the user